d/vibed/config.yaml

```diff
@@ -1,3 +1,3 @@
 framework:
   website: vibed.org
-  version: 0.9
+  version: 0.10
```

Comment:
```suggestion
  version: '0.10'
```

because of parsing

otherwise it will be displayed 0.1

Reply:
Fixed